<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentWindow</class>
 <widget class="QWidget" name="PaymentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>356</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="payButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>300</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Pay</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_number">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>351</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="inputMask">
    <string>0000 0000 0000 0000</string>
   </property>
   <property name="text">
    <string>   </string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>351</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>220</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Expiry date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_date">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>240</y>
     <width>181</width>
     <height>24</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="inputMask">
    <string>00/00</string>
   </property>
   <property name="placeholderText">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Card number</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>160</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Name on card</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_cvv">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>240</y>
     <width>101</width>
     <height>24</height>
    </rect>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>220</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Security code</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>payButton</sender>
   <signal>clicked()</signal>
   <receiver>PaymentWindow</receiver>
   <slot>openPaymentStatus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>297</x>
     <y>316</y>
    </hint>
    <hint type="destinationlabel">
     <x>369</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>openPaymentStatus()</slot>
 </slots>
</ui>
